bin/browserify-ftw.js

```diff
@@ -71,9 +74,7 @@ try {
 }
 
 try {
-  options = require(fullConfig);
-  // Poor man's validation
-  if (!options.quote || !options.style) throw new Error(fullConfig + 'is not a valid refactor config');
+  options = defaults(require(fullConfig), defaultOptions);
```

Comment:
Ok, where should I move it? The thing require comes from the user option `-c`, so I can't do it ebfore parsing is complete.